templates/foreman/template.yml

```diff
@@ -0,0 +1,15 @@
+skeleton_path: ../skeleton.json
+skeleton_index_pattern_path: ../skeleton-index-pattern.json
+
+elasticsearch_template:
+  name: org.foreman.viaq-cdm
+  index_pattern: "project.foreman-*"
```

Comment:
What is your intended index name format?  e.g. `project.foreman-$clustername.*`  That is - what comes after the `foreman-`?

Reply:
In the ovirt case, they use `project.ovirt-metrics-*` for their metrics/monitoring data, and `project.ovirt-logs-*` for their logging data.  The `*` part is for the node type e.g. `ovirt-metrics-engine` is for the ovirt engine - perhaps you could do something similar for foreman - note that the part after `project` corresponds to an OpenShift project/namespace name - so for example in the ovirt case, they have to create `ovirt-metrics-engine` and `ovirt-logs-engine` projects.

For now, how about just `project.foreman-main` and create an OpenShift project `foreman-main`

Reply:
Let me change this, so the end result in rsyslog.conf would be:

    searchIndex="project.foreman-logs.2018.11.07"

Is that correct?

Reply:
yes, looks good - so the index_pattern can remain `project.foreman-*`